src/main/java/org/zalando/nakadi/service/EventStream.java

```diff
@@ -11,13 +12,16 @@
 
 import java.io.IOException;
 import java.io.OutputStream;
+import java.io.OutputStreamWriter;
+import java.io.Writer;
 import java.nio.charset.Charset;
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.function.Function;
 import java.util.stream.Collectors;
+
```

Comment:
@lhotari I have the same setup (using [square's style guide](https://github.com/square/java-code-styles)).  I'm not sure the project has a formatting guideline, so I just try to remove/reduce incidental changes.